learn-css-grid-by-building-a-magazine/css/style620.css

```diff
@@ -172,5 +172,13 @@ hr {
 }
 
 /* 
-Next Media queries
+Well done, you've made it so far!
```

Comment:
/*
Well done, you've made it so far!
Next, let's style our magazine so it will look great on every device. 
First, you need to write a media query to style elements when the screen is a certain width.
Lets start with `max-width 720px`. Add this code below:
```
@media only screen and(max-width: 720px) {
}
```